Sources/Common/Background Queue/ApiSyncQueueRunner.swift

```diff
@@ -30,10 +38,21 @@ open class ApiSyncQueueRunner {
     }
 
     public func performHttpRequest(
-        params: HttpRequestParams,
+        endpoint: CIOApiEndpoint,
+        requestBody: Data?,
```

Comment:
This file got modified to make the queue runner classes within each module simpler. See those changes in other files in this PR.